### PR TITLE
Doc: Adding PuppyGraph to the vendor list

### PR DESCRIPTION
**Pull request by FreeMem17**
This PR adds to the vendor page a section for PuppyGraph.

The paragraphs and links to be added contain more details about PuppyGraph.

### CODE REVIEW

File: landing-page/content/common/vendors.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ With Dremio, an organization can easily build and manage a data lakehouse in whi
 
 IOMETE is a fully-managed ready to use, batteries included Data Platform. IOMETE optimizes clustering, compaction, and access control to Apache Iceberg tables. Customer data remains on customer's account to prevent vendor lock-in. The core of IOMETE platform is a serverless Lakehouse that leverages Apache Iceberg as its core table format. IOMETE platform also includes Serverless Spark, an SQL Editor, A Data Catalog, and granular data access control. IOMETE supports Hybrid-multi-cloud setups. 
 
+### [PuppyGraph](https://puppygraph.com)
+
+PuppyGraph is a cloud-native graph analytics engine that enables users to query one or more relational data stores as a unified graph model. This eliminates the overhead of deploying and maintaining a siloed graph database system, with no ETL required. PuppyGraph’s native Apache Iceberg integration is the easiest and most performant way to add native graph capabilities to your existing data lake.
```
Comment (danielcweeks) [inline landing-page/content/common/vendors.md:61]: > easiest and most performant
I think we want to keep comparative statements like this out of the description

Suggested alternative text:
> PuppyGraph’s native Apache Iceberg integration adds native graph capabilities to your existing data lake.

File: landing-page/content/common/vendors.md [hunk 2]
```diff
@@ -56,6 +56,16 @@ With Dremio, an organization can easily build and manage a data lakehouse in whi
...
+
+Deploy. Connect. Query. All in as little as 10 minutes.
```
Comment (danielcweeks) [inline landing-page/content/common/vendors.md:63]: I would suggest removing this as it's more "marketing" as opposed to "descriptive".

File: landing-page/content/common/vendors.md [hunk 3]
```diff
@@ -56,6 +56,16 @@ With Dremio, an organization can easily build and manage a data lakehouse in whi
...
+
+For more information, please see our:
+* [Iceberg deployment instructions](https://docs.puppygraph.com/user-manual/getting-started/iceberg)
```
Comment (danielcweeks) [inline landing-page/content/common/vendors.md:66]: I would link this inline with `Apache Iceberg integration` above and remove the image links.

Comment (FreeMem17) [reply landing-page/content/common/vendors.md]: Sounds good, updated accordingly.

File: landing-page/content/common/vendors.md (hunk 1 above)
Comment (FreeMem17) [reply landing-page/content/common/vendors.md]: Thanks for the suggestion. I avoid comparative or superlative adjectives in the paragraph now. Could you review the updated text and see if that would work?

**Review by danielcweeks — APPROVED** [review]

**Pull request merged**